src/ui/control/geolocate_control.js

```diff
@@ -451,14 +451,14 @@ class GeolocateControl extends Evented {
     */
     trigger() {
         if (!this._setup) {
-            warnOnce('Geolocate control triggered before added to a map');
+            warnOnce('GeolocateControl triggered before added to a map');
```

Comment:
This is failing a unit tests at _geolocate.test.js_. Needs an update to the expected text.